### ISSUE TITLE
NoMethodError: undefined method `join' for nil:NilClass

### ISSUE THREAD

**Issue by rohitn** [gshutler/useragent#32]
```ruby
$ irb
2.2.2 :001 > require 'useragent'
 => true
2.2.2 :002 > ua = UserAgent.parse("Mozilla/5.0 SHC; SHC-Unit-04973; SHC-HTS; SHC-KIOSK; SHC-MAC-0000 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2486.0 Safari/537.36 Edge/13.10586")
 => [#<UserAgent:0x007fbf5ca19960 @product="Mozilla", @version=#<UserAgent::Version 5.0>, @comment=nil>, #<UserAgent:0x007fbf5ca194d8 @product="SHC;", @version=#<UserAgent::Version >, @comment=nil>, #<UserAgent:0x007fbf5ca19230 @product="SHC-Unit-04973;", @version=#<UserAgent::Version >, @comment=nil>, #<UserAgent:0x007fbf5ca18f88 @product="SHC-HTS;", @version=#<UserAgent::Version >, @comment=nil>, #<UserAgent:0x007fbf5ca18ce0 @product="SHC-KIOSK;", @version=#<UserAgent::Version >, @comment=nil>, #<UserAgent:0x007fbf5ca18a10 @product="SHC-MAC-0000", @version=#<UserAgent::Version >, @comment=["Windows NT 10.0", "Win64", "x64"]>, #<UserAgent:0x007fbf5ca18678 @product="AppleWebKit", @version=#<UserAgent::Version 537.36>, @comment=["KHTML, like Gecko"]>, #<UserAgent:0x007fbf5ca18218 @product="Chrome", @version=#<UserAgent::Version 46.0.2486.0>, @comment=nil>, #<UserAgent:0x007fbf5ca23cf8 @product="Safari", @version=#<UserAgent::Version 537.36>, @comment=nil>, #<UserAgent:0x007fbf5ca23960 @product="Edge", @version=#<UserAgent::Version 13.10586>, @comment=nil>]
2.2.2 :003 > ua.os
NoMethodError: undefined method `join' for nil:NilClass
	from /Users/rohit/.rvm/gems/ruby-2.2.2/gems/useragent-0.16.5/lib/user_agent/browsers/edge.rb:21:in `os'
```

**Comment by gshutler** [comment]
This is fixed in 0.16.6, thanks for the report.

**Comment by rohitn** [comment]
@gshutler Thanks for the quick fix.
